docs/book/standard-filters.md

```diff
@@ -287,7 +287,7 @@ Type Constant | Type String | True | False
 ---- | ---- | ---- | -----
 `Laminas\Filter\Boolean::TYPE_BOOLEAN` | `boolean` | `TRUE` | `FALSE`
 `Laminas\Filter\Boolean::TYPE_EMPTY_ARRAY` | `array` | | `[]`
-`Laminas\Filter\Boolean::TYPE_FALSE_STRING` | `false` | `'false'` (case insensitive) | `'true'` (case insensitive)
+`Laminas\Filter\Boolean::TYPE_FALSE_STRING` | `false` | `'true'` (case insensitive) | `'false'` (case insensitive)
```

Comment:
Patch looks OK from an implementation PoV, but I will be honest, I don't understand the diff :D

Reply:
According to the documentation `'false'` would evaluate to `true` and `'true'` to `false` when applying filter. Now it displays that `'false'` evaluates to `false` when applying the filter :P.

Reply:
Ahhhh, it's a table, never mind -.- 

Thanks, I am not used to reading docs :P